payment-request/payment-request-abort-method.https.html

```diff
@@ -70,7 +70,7 @@
   // The request is now "closed", so...
   await promise_rejects(t, "InvalidStateError", request.abort());
   await promise_rejects(t, "AbortError", acceptPromise);
-});
+}, "aborting a request before it is shown doesn't prevent it from being shown later.");
```

Comment:
`aborting` is lowercase while every other description in the file started with uppercase.

heh, joking, you don't have to change it just messing with you for my first review request 😝

Reply:
No, consistency is good. Thanks for pointing that out.